airflow/www/static/css/tree.css

```diff
@@ -67,6 +67,10 @@ rect.shutdown {
   fill: blue;
 }
 
+rect.restarting {
+  fill: violet;
```

Comment:
We should add this state in https://github.com/apache/airflow/blob/c384f9b0f509bab704a70380465be18754800a52/airflow/settings.py#L83-L95

Reply:
Thanks @kaxil. I don't think that's necessary. Actually, the color is already hardcode in `state_color`, and then `STATE_COLORS` is used to override the values in `state_color` on this line:
https://github.com/apache/airflow/blob/c384f9b0f509bab704a70380465be18754800a52/airflow/utils/state.py#L117
Some states are already missing from `STATE_COLORS`, such as `NONE` and `SHUTDOWN`. I think we should either deprecate `STATE_COLORS` or auto-populate it from `state_color`. But maybe that's for a different PR.